slpfaucet.ts

```diff
@@ -4,8 +4,8 @@ import { GrpcClient } from "grpc-bchrpc-node";
 import * as slpjs from "slpjs";
 import { BchdNetwork, BchdValidator, Utils } from "slpjs";
 
-const bitbox = new BITBOX();
-const client = new GrpcClient({url: "bchd.ny1.simpleledger.io" });
+const bitbox = new BITBOX({restURL: 'https://bchd.fountainhead.cash'});
```

Comment:
We aren't using any of the network methods in BITBOX sdk, so we don't meed to use `restURL` parameter.